floss/main.py

```diff
@@ -207,6 +207,98 @@ def make_parser():
+    # DEBUG: merge_candidates: Function at 0x00401500 is new, adding
+    logging.getLogger("floss.identification_manager.IdentificationManager").setLevel(log_level)
+
+    # ignore messages like:
+    # WARNING: get_caller_vas: unknown caller function: 0x403441
+    # DEBUG: get_all_function_contexts: Getting function context for function at 0x00401500...
+    logging.getLogger("floss.function_argument_getter.FunctionArgumentGetter").setLevel(log_level)
+
+    # ignore messages like:
+    # DEBUG: Emulating function at 0x004017A9 called at 0x00401644, return address: 0x00401649
+    logging.getLogger("floss").setLevel(log_level)
+
+    # ignore messages like:
+    # DEBUG: extracting stackstrings at checkpoint: 0x4048dd stacksize: 0x58
+    logging.getLogger("floss.stackstrings").setLevel(log_level)
+
+    # ignore messages like:
+    # WARNING:plugins.arithmetic_plugin.XORPlugin:identify: Invalid instruction encountered in basic block, skipping: 0x4a0637
+    logging.getLogger("floss.plugins.arithmetic_plugin.XORPlugin").setLevel(log_level)
+    logging.getLogger("floss.plugins.arithmetic_plugin.ShiftPlugin").setLevel(log_level)
+
+    # ignore messages like:
+    # DEBUG: identify: Identified WSAStartup_00401476 at VA 0x00401476
+    logging.getLogger("floss.plugins.library_function_plugin.FunctionIsLibraryPlugin").setLevel(log_level)
+
+    # ignore messages like:
+    # DEBUG: identify: Function at 0x00401500: Cross references to: 2
+    logging.getLogger("floss.plugins.function_meta_data_plugin.FunctionCrossReferencesToPlugin").setLevel(log_level)
+
+    # ignore messages like:
+    # DEBUG: identify: Function at 0x00401FFF: Number of arguments: 3
+    logging.getLogger("floss.plugins.function_meta_data_plugin.FunctionArgumentCountPlugin").setLevel(log_level)
+
+    # ignore messages like:
+    # DEBUG: get_meta_data: Function at 0x00401470 has meta data: Thunk: ws2_32.WSACleanup
+    logging.getLogger("floss.plugins.function_meta_data_plugin.FunctionIsThunkPlugin").setLevel(log_level)
+
+    # ignore messages like:
+    # DEBUG: get_meta_data: Function at 0x00401000 has meta data: BlockCount: 7
+    logging.getLogger("floss.plugins.function_meta_data_plugin.FunctionBlockCountPlugin").setLevel(log_level)
+
+    # ignore messages like:
+    # DEBUG: get_meta_data: Function at 0x00401000 has meta data: InstructionCount: 60
+    logging.getLogger("floss.plugins.function_meta_data_plugin.FunctionInstructionCountPlugin").setLevel(log_level)
+
+    # ignore messages like:
+    # DEBUG: get_meta_data: Function at 0x00401000 has meta data: Size: 177
+    logging.getLogger("floss.plugins.function_meta_data_plugin.FunctionSizePlugin").setLevel(log_level)
+
+    # ignore messages like:
+    # DEBUG: identify: suspicious MOV instruction at 0x00401017 in function 0x00401000: mov byte [edx],al
+    logging.getLogger("floss.plugins.mov_plugin.MovPlugin").setLevel(log_level)
+        
 
 def set_logging_level(should_debug=False, should_verbose=False):
```

Comment:
let's change the name of this routine, as described above.

Reply:
Yep, done.